fugue-aws/src/main/java/com/symphony/oss/fugue/aws/sqs/SqsQueueSender.java

```diff
@@ -63,7 +65,14 @@
   {
     sqsClient_     = sqsClient;
     
-    queueUrl_ = sqsClient_.getQueueUrl(queueName).getQueueUrl();
+    try
+    {
+      queueUrl_ = new URL(queueName).toString();
+    }
+    catch (MalformedURLException e)
+    {
+      queueUrl_ = sqsClient_.getQueueUrl(queueName).getQueueUrl();
```

Comment:
What's happening here? Does queueName sometimes contain a URL?

This smells bad, I don't like relying on an exception like this and the field should still be final so move the init logic into a static method please.

Reply:
yes originally I used a flag, in case of direct fetch the queueUrl is already provided from the server, the exception case is on the server itself and it's the classic case it needs to be resolved by the sqsclient, 
I'll revert